var/spack/repos/builtin/packages/at-spi2-core/package.py

```diff
@@ -21,7 +21,7 @@ class AtSpi2Core(MesonPackage):
     version('2.36.0', sha256='88da57de0a7e3c60bc341a974a80fdba091612db3547c410d6deab039ca5c05a')
     version('2.28.0', sha256='42a2487ab11ce43c288e73b2668ef8b1ab40a0e2b4f94e80fca04ad27b6f1c87')
 
-    depends_on('meson@0.46.0:', type='build')
+    depends_on('meson@0.46.0:', type='build', when='@2.36.0:')
```

Comment:
Can you give some context? Without this v2.28.0 will just depend on `meson` without a minimum version. Is it solving some issues?

Reply:
It does not in itself solves the problem (we overwrote `at-spi2-core/package.py` with an older version to prevent the meson dependency), but it _documents_ the dependency. If spack ever gets around to "solve" multi-buildtool dependencies in a clean way

Reply:
> It does not in itself solves the problem (we overwrote at-spi2-core/package.py with an older version to prevent the meson dependency)

I think I don't understand what you're saying here...

> but it documents the dependency

What I mean is that before v2.28.0 had a requirement that is now being lifted. So it may be documenting the _lack of a constraint_. I was wondering why this is necessary.

Reply:
@healther